tests/modules/nf-core/cellranger/mkvdjref/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: cellranger mkvdjref test_cellranger_mkvdjref
+  command: nextflow run ./tests/modules/nf-core/cellranger/mkvdjref -entry test_cellranger_mkvdjref -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/cellranger/mkvdjref/nextflow.config
+  tags:
+    - cellranger/mkvdjref
+    - cellranger
+  files:
+    - path: output/cellranger/homo_sapiens_chr22_reference/fasta/regions.fa
```

Comment:
Can add a `contains` with a string opf teh fasta entry

Reply:
no that's not possible

the file is empty since the nf-core/test-datasets chr22 FASTQs don't have VDJ sequences

but the MD5 sum should be stable in that case

Reply:
MD5 sum is indeed stable but empty files enrage the linter, so current approach is to just check for file existence at `path:`